advntr/pairwise_aln_generator.py

```diff
@@ -343,12 +342,17 @@ def _generate_pairwise_aln(log_file, aln_outfile, ref_vntrs, vid_list=None, sort
                             read_id = ""
                             read_source = ""
                         else:
-                            read_source = split_line[7]  # either "MAPPED" or "UNMAPPED"
+                            read_source = split_line[6]  # either "MAPPED" or "UNMAPPED"
 
                         visited = line[line.index('[') + 1:-2]
                         visited_states = [item[1:-1] for item in visited.split(', ')]
                     else:
-                        sequence = line[30:].strip()
+                        # Make sure it is a sequence string
+                        tentative_sequence = line[30:].strip()
+                        # Make sure there are only 4 characters are ACTG
+                        # Otherwise, it is an unrelated log line.
+                        if "".join(sorted(set(list(tentative_sequence)))) == "ACGT":
```

Comment:
what if the sequence is composed of one character?

Reply:
I fixed the issue. I'm now checking if the sequence is a subset of the set("ACGT")